src/middleware/navigation/update/has.beneficial.owners.or.managing.officers.update.middleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { logger } from '../../../utils/logger';
+import { UPDATE_BENEFICIAL_OWNER_TYPE_URL } from '../../../config';
+import { getApplicationData } from "../../../utils/application.data";
+import { checkBOsOrMOsDetailsEnteredUpdate, NavigationErrorMessage } from '../check.condition';
+
+export const hasBOsOrMOsUpdate = (req: Request, res: Response, next: NextFunction): void => {
+  try {
+    if ( !checkBOsOrMOsDetailsEnteredUpdate(getApplicationData(req.session)) ) {
```

Comment:
have you "make lint" on this?

Reply:
Passes the lint check okay, although whitespace has been removed